EthOn.rdf

```diff
@@ -1633,8 +1633,9 @@
         <owl:hasKey rdf:parseType="Collection">
             <rdf:Description rdf:about="http://ethon.consensys.net/address"/>
         </owl:hasKey>
+        <simpleDefinition>Accounts are the actors in Ethereum. There are two main types of Accounts: External Accounts (often referred to as "wallet" or simply "account") and Contract Accounts (usually just called "contract" or "smart contract"). An External Account is what you need to use Ethereum. It is usually protected by a password. If you have the password you can send Ether from your Account to another or use a contract. It has an Ethereum address (0x...) by which it is identified.</simpleDefinition>
```

Comment:
I think the best way to clarify this would be to remove the first sentence all together. It's not really good, like you say, and a bit ambiguous.
Don't think the simple comment is too long. Doesn't have to be short to be simple.